packages/opentelemetry-node/src/config.ts

```diff
@@ -27,20 +27,24 @@ export interface NodeTracerConfig extends BasicTracerConfig {
 
 /** List of all default supported plugins */
 export const DEFAULT_INSTRUMENTATION_PLUGINS: Plugins = {
-  http: {
+  dns: {
     enabled: true,
-    path: '@opentelemetry/plugin-http',
+    path: '@opentelemetry/plugin-dns',
   },
   grpc: {
     enabled: true,
     path: '@opentelemetry/plugin-grpc',
   },
+  http: {
+    enabled: true,
+    path: '@opentelemetry/plugin-http',
+  },
   https: {
     enabled: true,
     path: '@opentelemetry/plugin-https',
   },
-  dns: {
+  mysql: {
```

Comment:
Could you please remove WIP from mysql plugin from https://github.com/open-telemetry/opentelemetry-js#node-plugins?